### DISCUSSION TITLE
Error while deploying

### DISCUSSION THREAD

**Discussion by Wonderdog321** [vercel/community#2630] [Help]
### Summary

So I checked my vercel account and its set to 18.*.* and I've downgraded my node to 18.0.0 yet I still keep having issues with it poping these errors.

### Steps to Reproduce

`<html>
<body>

Running build in Cleveland, USA (East) – cle1
--
03:42:32.934 | Cloning github.com/**********/*******(Branch: master, Commit: *****)
03:42:32.941 | Skipping build cache, deployment was triggered without cache.
03:42:33.285 | Cloning completed: 350.586ms
03:42:33.458 | Running "vercel build"
03:42:34.000 | Vercel CLI 29.3.6
03:42:34.590 | Installing dependencies...
03:42:41.636 |  
03:42:41.636 | added 199 packages in 7s
03:42:41.636 |  
03:42:41.636 | 15 packages are looking for funding
03:42:41.636 | run `npm fund` for details
03:42:41.659 | node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by node)
03:42:41.660 | node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by node)
03:42:41.669 | Error: Command "CI='' npm start" exited with 1
03:42:41.891 | BUILD_UTILS_SPAWN_1: Command "CI='' npm start" exited with 1

</body>
</html>`

I'm not sure what else to do. I have it set to the lowest supported by vercel.

**Comment by z4nr34l** [comment]
Could you bring us more details?
I consider as helpful those informations:
1. Repository Access;
2. Vercel build logs (from deployment with this issue);
3. Runtime logs from problematic request;

The more - the better for receiving quick help.

You can also try to reproduce this problem on free [CodeSandbox](https://codesandbox.io/) and post URL here when it's done.
